### PR TITLE
judge: fix the mypy error in CI

### PR DESCRIPTION
**Pull request by JasonLovesDoggo**
See https://github.com/DMOJ/judge-server/actions/runs/10466584470/job/28983707605

(https://github.com/mcpt/wlmoj-judge/actions/runs/10975887920/job/30476034151) for a successful run

### CODE REVIEW

File: dmoj/executors/asm_executor.py [hunk 1]
```diff
@@ -239,7 +239,11 @@ class UnknownPlatformMixin(ASMExecutor):
...
-    [cls for cls in (PlatformX86Mixin, PlatformX64Mixin, PlatformARMMixin, PlatformARM64Mixin) if cls.abi == NATIVE_ABI]
-    or [UnknownPlatformMixin]
-)[0]
+NativeMixin: Any = next(
+    (
+        cls
+        for cls in (PlatformX86Mixin, PlatformX64Mixin, PlatformARMMixin, PlatformARM64Mixin)
+        if cls.abi == NATIVE_ABI
+    ),
+    UnknownPlatformMixin,
```
Comment (kiritofeng) [inline dmoj/executors/asm_executor.py:248]: Is this comma removable?

Comment (JasonLovesDoggo) [reply dmoj/executors/asm_executor.py:248]: No you do not 👍🏼

**Review by kiritofeng — APPROVED** [review]
LGTM, thanks!

**Comment by JasonLovesDoggo** [comment]
Ahh, found why that was added.... yeah black likes it 🤷🏼

File: dmoj/executors/asm_executor.py (hunk 1 above)
Comment (JasonLovesDoggo) [reply dmoj/executors/asm_executor.py:248]: (update: Black likes it)

**Pull request merged**